nastaveni/hlasky/nastaveni-hlasky-subst.php

```diff
@@ -25,7 +25,7 @@
     'rychloregMail'                         =>
         'Ahoj,
 
-děkujeme, že ses letos zúčastnil{a} GameConu. Kliknutím na odkaz níže potvrdíš registraci na web a můžeš si nastavit přezdívku a heslo, pokud chceš používat web a třeba přijet příští rok. (Pokud by ses registroval{a} na web později, musel{a} by sis nechat vygenerovat heslo znova)
+děkujeme, že se letos zúčastníš GameConu. Kliknutím na odkaz níže potvrdíš registraci na web a můžeš si nastavit přezdívku a heslo, pokud chceš používat web a třeba přijet i příští rok. (V případě pozdější registrace na web by bylo nutné nechat si vygenerovat heslo znova)
```

Comment:
Zde dochází ke změně času a tím i významu

Každopádně je to ale úplně jedno, neboť tato hláška se už efektivně nikde nepoužívá - rychloregistrace už nesbírají emaily ani nic neposílají. (plány do budoucna ale neznám)

Reply:
Pravda. Bude v opravě.